tests/vectors/protocol-definitions/social-media.json

```diff
@@ -24,20 +24,28 @@
     "image": {
       "allow": {
         "anyone": {
-          "to": ["write"]
+          "to": ["read", "write"]
         }
       },
       "records": {
         "caption": {
           "allow": {
             "author": {
-                "of": "image",
-                "to": ["write"]
+              "of": "image",
+              "to": ["write"]
+            },
+            "recipient": {
```

Comment:
Nit: like earlier comment, `recipient` here (and line 49) feels out of place, `anyone` seems to be more appropriate. If you need to test `recipient` rule, consider using/extending `dex.json` or `email.json` (coming in the other PR) for a more realistic (thus easier-to-understand) scenario.